libraries/heft-config-file/src/test/ConfigurationFile.test.ts

```diff
@@ -524,13 +524,9 @@ describe('ConfigurationFile', () => {
           'config.schema.json'
         )
       });
-      try {
-        expect(
-          await configFileLoader.tryLoadConfigurationFileForProjectAsync(terminal, __dirname)
-        ).toBeUndefined();
-      } catch (e) {
-        fail();
-      }
+      expect(
+        await configFileLoader.tryLoadConfigurationFileForProjectAsync(terminal, __dirname)
+      ).toBeUndefined();
```

Comment:
impl note: this change was not required for tests to pass, it's just a small enhancement to the test output if this test fails (show the exception you are getting rather than failing with no information).